exercises/atbash-cipher/src/Atbash.hs

```diff
@@ -1,7 +1,7 @@
 module Atbash (decode, encode) where
 
 decode :: String -> String
-decode phrase = error "You need to implement this function."
+decode xs = error "You need to implement this function."
```

Comment:
`xs` is great when we don't know anything about the content, so there is not way to give more information. Which is not the case.

I think that `phrase` conveys meaning, but not necessarily the correct one:

> [phrase](http://www.thefreedictionary.com/phrase) n.
> A sequence of words that have meaning, especially when forming part of a sentence.

In the context of `decode`, the argument is encoded, so I don't think it fits the definition very sell, and that's why I prefer `xs` over `phrase` here.

Of course, in a cryptography-context, there are more descriptive names, like [plainText](https://en.wikipedia.org/wiki/Plaintext) and [cipherText](cipherText).